employee-management-client/src/app/shared/client.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { HttpClient } from '@angular/common/http';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ClientService {
+
+  constructor(private _http: HttpClient) { }
+
+
+  form: FormGroup = new FormGroup({
```

Comment:
As this form group specific to client component, rather than declaring it in service, keep component specific code in component itself. Only shared function/variables & API calls should keep in service file